Gaijinizer patch/CommonEvents_440.txt

```diff
@@ -87,45 +87,52 @@ But here I forget that. Let's enjoy each other
 気がつくと彼女の腰に手を回し、彼女の豊満な胸に
 顔を埋めようと身をかがめている…
 > CONTEXT: CommonEvents_440/79/Dialogue 
-My reason was giving up thinking about it a little further
-... When I notice, I put my hand on her waist and on her
-plump chest. I'm crouching to fill my face ...#MTLed
+My mind has abandoned any intention of trying to stay a
+step ahead of her. Before I know it, I have a hand around
+her waist as I bend down to try and bury my face in her
+plump chest.#Ron
 > END STRING
```

Comment:
not sure if we change the tense on the sentence here...
```
My mind had abandoned any intention of trying to stay a
step ahead of her. Before I knew it, I already had a hand
around her waist as I bent down to try and bury my face in
her plump chest.
```

Slight localization thought: maybe we could word the first sentence like this? Not sure if it matches the intention of the JP...
```
My body was already one step ahead of my thoughts...
```

Need to chat a bit on it.

Reply:
The way I interpreted that first line of "僕の理性は、一足先にもう考える事を放棄していた" was his mind had stopped trying to think two moves ahead of the situation to avoid being bamboozled by a demon.

Reply:
Cool, then just make the tense changes and it should be good.